Automattic/wp-calypso
# Launchpad: Remove useLaunchpad hook from data

Related to #76364

## Proposed Changes

* Removes `useLaunchpad` hook from calypso/data, it's been moved to `@automattic/data-stores` and all references have been pointed to the data-stores instance.

## Testing Instructions
* Switch to this PR
* Create a new free site from `/start` with the Write & Publish intent
* Skip to dashboard to see Launchpad
* Ensure Launchpad still works as expected and there are no errors in the console

## Pre-merge Checklist
